sim/src/mechanics/driving.rs

```diff
@@ -248,7 +248,25 @@ impl DrivingSimState {
         transit: &mut TransitSimState,
     ) -> bool {
         match car.state {
-            CarState::Crossing(_, _) => {
+            CarState::Crossing(time_int, dist_int) => {
+                let time_cross = now - time_int.start;
+                let avg_speed = Speed::from_dist_time(dist_int.length(), time_cross);
```

Comment:
I got a NaN here when simulating the full weekday on the lakeslice map. If `time_cross == Duration::ZERO`, we should skip emitting an event here. I'm not totally sure how this happened.

Reply:
Weird, but done